utils/encryption.js

```diff
@@ -0,0 +1,16 @@
+const bcrypt = require('bcrypt')
+const saltRounds = 10
+
+const hashPassword = async (userInputPassword) => {
+    const hashed = await bcrypt.hash(userInputPassword, saltRounds);
```

Comment:
This is perfectly fine so you don't need to change - you could return the hash function - as it is an async function awaiting hashPassword will await that 

What you have is pretty explicit though which is also good
```js
const hashPassword = async (userInputPassword) => {
  return bcrypt.hash(userInputPassword, saltRounds)
}
```